docs/patching-pega-deployment.md

```diff
@@ -0,0 +1,146 @@
+
+5. Apply the patch by using the `helm upgrade-deploy` command as directed in the deployment section - [Deploy Pega Platform using the command line](#patch-your-pega-platform-deployment-using-the-command-line).
+
+6. Re-enable rule creation in your current deployment. Depending on your version of Pega Platform, the steps are slightly different, but the same steps used in on-premises systems:
+
+   - For 8.4 and later, see [For systems configured for High Availability: Reverting the cluster settings](https://community.pega.com/knowledgebase/articles/keeping-current-pega/85/systems-configured-high-availability-reverting-cluster-settings).
+   - For 8.2 or 8.3, see [For systems configured for High Availability: Reverting the cluster settings](https://community.pega.com/knowledgebase/articles/keeping-current-pega/85/systems-configured-high-availability-reverting-cluster-settings).
+
+## Assumptions and prerequisites
+
+The process to patching your deployment assumes:
+
+- Your Kubernetes environment has not changed and you are using the same Pega charts with which you originally deployed.
+
+- You download the latest Pega docker images in the minor release stream with which you initially deployed.
+
+- Your original deployment used Pega Platform 8.2.1 or later.
+
+## Downloading your images for the patch process – 20 minutes
+
+Patching the Pega software in your deployment requires the use of the required Docker images built from the same release stream for the software running in your deployment. To do this, you must download and make all three of the new images available in your preferred Docker registry. For details, see [Downloading Docker images for your deployment](https://github.com/pegasystems/pega-helm-charts#downloading-docker-images-for-your-deployment).
+
+See the next section to properly reference these new images in your Pega Helm chart.
+
+## Applying a zero-downtime Pega Platform patch using Helm charts – 120 minutes
+
+To keep your system current with a patch, you apply a patch to the Pega Platform software in your existing deployment by using Helm, you must customize your existing `pega` Helm chart with the specific, required or settings. This includes referencing the latest Pega-provided Docker images that are available for your release.
+
+The Pega patch application process takes at most 120 minutes total.
+
+To complete a zero downtime patch, you must configure the following settings in your existing Pega configuration files for your Pega Platform deployment:
+
+- Specify action.execute: upgrade-deploy to invoke the zero-downtime patch process.
+- Specify the schema name or names that will be upgraded. Since the process involves migrating rules to and from each schema (jdbc.connectionProperties.rulesSchema: "YOUR_RULES_SCHEMA" and jdbc.connectionProperties.dataSchema: "YOUR_DATA_SCHEMA").
```

Comment:
please correct the typo here.. its 'jdbc.rulesschema' and 'jdbc.dataschema'. Please remove the work 'connectionProperties' in between